mono/metadata/icall.c

```diff
@@ -8589,9 +8589,14 @@ ves_icall_System_Runtime_InteropServices_Marshal_Prelink (MonoReflectionMethodHa
 	prelink_method (MONO_HANDLE_GETVAL (method, method), error);
 }
 
+void
+ves_icall_System_Runtime_InteropServices_Marshal_PrelinkAll (MonoReflectionTypeHandle type, MonoError *error);
```

Comment:
Why is this better than putting an ifdef around the function ?

Reply:
It lets you compile for netcore, but still verify that you didn't break compilation of non-netcore. And vice versa.